styles.css

```diff
@@ -769,4 +865,37 @@ a {
     color: #5b5957;
     background-color: #f4f1f0;
   }
+
+  #modal-main {
+    margin-top: 5vw;
+  }
+
+  #modal-main #image {
+    margin-right: 1.8vw;
+    width: 50%;
+    height: auto;
+  }
+
+  .modal-header {
+    margin-bottom: 0.83vw;
+  }
+
+  .modal {
+    padding: 2vw;
+    right: 5vw;
+    left: 5vw;
+    position: fixed;
+    top: 2vw;
+    transition: 200ms ease-in-out;
+    border: 1px solid #ebecf0;
+    border-radius: 10px;
+    z-index: 50;
+    background-color: white;
+    width: 90vw;
+    height: auto;
+  }
+
+  #mod-btn .modal-icon {
+    margin-left: 1vw;
+  }
```

Comment:
Please make sure to stick to the Figma design template as much as possible. Adjust the size of your `icon` so your button text is able to align in the center to create a good user experience.
Figma:
<img width="774" alt="Screenshot 2023-04-20 at 20 17 07" src="https://user-images.githubusercontent.com/107192076/233478349-8e9ad62c-3107-45a6-80c5-8858d4af018f.png">
Your design:
<img width="862" alt="Screenshot 2023-04-20 at 20 09 32" src="https://user-images.githubusercontent.com/107192076/233478428-6e5a476b-629d-45e8-932c-47e00890c8b5.png">